test/CMakeLists.txt

```diff
@@ -38,7 +38,7 @@ file(DOWNLOAD https://gitlab.cern.ch/VecGeom/VecGeom/raw/v1.2.0/persistency/gdml
 #----------------------------------------------------------------------------#
 # Detailed tests
 #----------------------------------------------------------------------------#
-add_subdirectory(testField)
+# add_subdirectory(testField)
```

Comment:
Does this not compile or work anymore? Or can this be activated again? What is the status of this?